switchboard/switchboard/src/server.rs

```diff
@@ -102,16 +102,19 @@ pub async fn database_from_config(cfg: &Config) -> miette::Result<PgPool> {
 pub async fn serve(cmd: ServeCommand) -> miette::Result<()> {
     let config_path = &cmd.cfg;
 
-    // Tracing init
-
-    // console_subscriber::init();
-    tracing_subscriber::fmt::init();
-
     // Load configuration
 
     // TODO: overlayed configuration
     let config = crate::cfg::load_config(config_path)?;
 
+    // Tracing init
+
+    if config.server.tokio_console_server {
+        console_subscriber::init();
+    } else {
+        tracing_subscriber::fmt::init();
+    }
+
```

Comment:
Will this prevent log messages from being printed on the command line when the console server is enabled?

Reply:
`console_subscriber` is basically `tracing_subscriber::fmt`, but with an extra layer that intercepts `tokio`'s internal structured `tracing` output (and only `tokio`'s), so switchboard's normal logging will work fine, and will still respect the `RUST_LOG` environmental variable, the value of which will not interfere with `tokio-console`'s functionality.